src/database/migrations/20240513094933-create-comments-table.js

```diff
@@ -30,8 +30,8 @@ module.exports = {
 			},
 			status: {
 				allowNull: false,
-				type: Sequelize.ENUM('OPEN', 'RESOLVED'),
-				defaultValue: 'OPEN',
+				type: Sequelize.ENUM('OPEN', 'RESOLVED', 'DRAFT'),
```

Comment:
We have to update the enum manually for this right? @priyanka-TL

Reply:
yes